views/update/do-you-want-to-make-oe-change.html

```diff
@@ -10,7 +16,6 @@
 
     {% include "includes/list/errors.html" %}
     <span class="govuk-caption-xl govuk-!-padding-bottom-1">{{ entity_name }} - {{ entity_number }}</span>
-    <h1 class="govuk-heading-xl govuk-!-margin-0">Do you need to make any changes to this overseas entity?</h1>
```

Comment:
Just for visibility, I asked Cory why this was removed and not updated with `{{ title }}` like everywhere else.

The legend in the radio button question below has `legend: { isPageHeading: true, text: title }`, which, visible from the diff, was not previously set, but now is on line 3, so this h1 is no longer needed.